server/src/tests/namedtuple.test.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Please add these test cases to the existing testAnalyzer.test.ts rather than creating a new test source file.